### PR TITLE
Update examples readmes according to 0.13

### PR DESCRIPTION
**Pull request by ivan-ochc**

### CODE REVIEW

**Comment by vitvakatu** [comment]
It isn't user-friendly, as they must create a config file themselves. I think the example config files must be provided.

File: examples/timestamping/timestamping.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+deadline_height: 10000
+supervisor_mode: "simple"
+
+artifacts:
+  time:
+    runtime: rust
+    name: "exonum-time:0.12.0"
+  timestamping:
+    runtime: rust
+    name: "exonum-timestamping:0.0.0"
```
Comment (alekseysidorov) [inline examples/timestamping/timestamping.yaml]: `exonum-cryptocurrency-advanced` has version `0.12.0`, but `exonum-timestamping` has `0.0.0` that looks a little bit inconsistent.

File: examples/cryptocurrency-advanced/README.md [hunk 2]
```diff
@@ -117,6 +118,12 @@ exonum-cryptocurrency-advanced run --node-config example/3/node.toml --db-path e
 exonum-cryptocurrency-advanced run --node-config example/4/node.toml --db-path example/4/db --public-api-address 0.0.0.0:8203 --master-key-pass pass
 ```
 
+Deploy cryptocurrency-advanced service:
+
+```sh
+python3 -m exonum_launcher -i ../cryptocurrency_advanced.yaml
```
Comment (aleksuss) [inline examples/cryptocurrency-advanced/README.md]: I suppose you need to describe the process of `exonum_launcher` module installing.

File: examples/timestamping/timestamping.yaml (hunk 1 above)
Comment (vitvakatu) [reply examples/timestamping/timestamping.yaml]: It is not config problem, but rather timestamping example problem. The version must be changed there as well

File: examples/cryptocurrency-advanced/README.md (hunk 2 above)
Comment (ivan-ochc) [reply examples/cryptocurrency-advanced/README.md]: There is a link to exonum-launcher above in the "Be sure you installed necessary packages:" section.

File: examples/cryptocurrency-advanced/README.md [hunk 3]
```diff
@@ -117,6 +118,19 @@ exonum-cryptocurrency-advanced run --node-config example/3/node.toml --db-path e
 exonum-cryptocurrency-advanced run --node-config example/4/node.toml --db-path example/4/db --public-api-address 0.0.0.0:8203 --master-key-pass pass
 ```
 
+Before service deploy make sure that you have pure python implementation of protobuf:
```
Comment (vitvakatu) [inline examples/cryptocurrency-advanced/README.md]: Interesting. @popzxc what is the reason? Shouldn't it be specified in the `exonum_launcher`?

Comment (ivan-ochc) [reply examples/cryptocurrency-advanced/README.md]: It has been specified in exonum-python-client.

Comment (vitvakatu) [reply examples/cryptocurrency-advanced/README.md]: I see now. Unfortunate problem :(

Comment (popzxc) [reply examples/cryptocurrency-advanced/README.md]: >what is the reason?

The reason is extremely "good" protobuf implementation for python: it ignores the `package` field set in files and thinks that two files with the same name are always belong to the same descriptor (and we have plenty of those, e.g. `service.proto`). The intended way (I asked about it in the protobuf repo) is to compile files with the same name providing the folders name as well, so they eventually will become different names.

Pure python implementation softens those restrictions, but they've said that they want to implement this behaviour for pure python library as well :cold_sweat: 

I'm currently thinking what can we do about it, but for now using the pure python implementation is the simplest approach we can get.

File: examples/timestamping/README.md [hunk 4]
```diff
@@ -111,6 +112,19 @@ exonum-timestamping run --node-config example/3/node.toml --db-path example/3/db
...
 ```
 
+Before service deploy make sure that you have pure python implementation of protobuf:
+
+```sh
+pip uninstall protobuf
+pip install --no-binary=protobuf protobuf
+```
+
+Deploy cryptocurrency-advanced service.
```
Comment (aleksuss) [inline examples/timestamping/README.md]:
```suggestion
Deploy timestamping service.
```

**Pull request merged**